packages/@pollyjs/core/tests/unit/test-helpers/mocha-test.js

```diff
@@ -34,21 +34,15 @@ describe('Unit | Test Helpers | mocha', function() {
   });
 
   it('should create a polly property and set recordingName', function() {
```

Comment:
The test on line 28 was the one that was now failing.  This was because the async was putting the [recording name assertion](packages/@pollyjs/core/src/utils/validators.js) on the next tick.